pyproject.toml

```diff
@@ -31,11 +31,13 @@ matplotlib = "^3.1.3"
 numpy = "^1.18.1"
 pandas = "<=1.0.3"
 pandas_datareader = "^0.8.1"
-pytz = "^2019.3"
+pytz = ">=2019.3"
 requests = "^2.23.0"
 scikit-learn = "^0.23.0"
 scipy = "^1.4.1"
 statsmodels = "^0.11.0"
+seaborn = "^0.11.0"
```

Comment:
Thanks for including seaborn. We'll need to do a check that no other package is needed at some point.

Reply:
All unittests are passing, so I guess we are OK.
Next step will be adding coverage report. But I keep it for another branch